elftools/dwarf/dwarfinfo.py

```diff
@@ -112,6 +117,84 @@ def has_debug_info(self):
         """
         return bool(self.debug_info_sec)
 
+    def get_DIE_from_lute(self, lutentry):
```

Comment:
Nit: Maybe rename to either `get_DIE_from_lut` or `get_DIE_from_lut_entry`?

Reply:
@eliben want me to fix this up and rebase?  (I'd choose lut_entry) .. then I can rebase @265 on top

Reply:
lut_entry SGTM